opslevel/resource_opslevel_service.go

```diff
@@ -62,13 +62,27 @@ func newServiceResourceModel(ctx context.Context, service opslevel.Service, give
 		Language:        OptionalStringValue(service.Language),
 		LifecycleAlias:  OptionalStringValue(service.Lifecycle.Alias),
 		Name:            RequiredStringValue(service.Name),
-		Note:            givenModel.Note,
-		Owner:           OptionalStringValue(givenModel.Owner.ValueString()),
-		Parent:          OptionalStringValue(givenModel.Parent.ValueString()),
+		Note:            OptionalStringValue(service.Note),
 		Product:         OptionalStringValue(service.Product),
 		TierAlias:       OptionalStringValue(service.Tier.Alias),
 	}
 
+	if string(service.Owner.Id) == "" {
+		serviceResourceModel.Owner = types.StringNull()
+	} else if string(service.Owner.Id) == givenModel.Owner.ValueString() || service.Owner.Alias == givenModel.Owner.ValueString() {
+		serviceResourceModel.Owner = givenModel.Owner
+	} else {
+		serviceResourceModel.Owner = OptionalStringValue(service.Owner.Alias)
```

Comment:
Should this be `alias` or should it be the `id` ?

I think historically we've prioritized adding the `id` to the state because thats stable and the `alias` is not because people can move them.

Reply:
That makes more sense. Changing this to Id